dit/ditexporter.py

```diff
@@ -87,10 +87,10 @@ def task(group, group_id, subgroup, subgroup_id, task, task_id, data):
         for note in notes:
             write('  * %s' % note)
 
-        if props:
+        if properties:
             write('  Properties:')
-        for prop in props:
-            write('  * %s: %s' % (prop['name'], prop['value']))
+        for prop in properties:
```

Comment:
use `prop_name` instead of `prop` to [keep consistency](https://github.com/filipelbc/dit/pull/126/files#diff-5284e71c5000e7320885ddf0eea55979R489).